### PR TITLE
Deprecate various

### PR DESCRIPTION
**Pull request by TonyBagnall**
### Reference Issues/PRs
Deprecates MrSEQL cython version in lines with 
https://github.com/alan-turing-institute/sktime/issues/1547
https://github.com/alan-turing-institute/sktime/issues/1538

#### What does this implement/fix? Explain your changes.

Deprecation for MrSEQL prior to the removal of cython

#### What should a reviewer concentrate their feedback on?
 
is this sufficient to indicate deprecation?

### CODE REVIEW

**Comment by MatthewMiddlehurst** [comment]
Tests can be excluded in tests/_config.py by adding them to EXCLUDED_TESTS.

Is the decorator necessary? Could we not just print a warning in the init or at the top of the function or similar?

**Comment by TonyBagnall** [comment]
We could just leave the comment, it's my preference, but would like consensus on it. I will ask core devs

**Comment by mloning** [comment]
You don't have to use the decorator, you can just raise a FutureWarning in the method/function, would that help?

**Comment by TonyBagnall** [comment]
> You don't have to use the decorator, you can just raise a FutureWarning in the method/function, would that help?

I'll try it, thanks, lets see if it still screw the tests

**Comment by TonyBagnall** [comment]
raising a future warning or using warn() doesnt work, the cython breaks

**Comment by MatthewMiddlehurst** [comment]
Are there only a few tests failing? Should be kept in mind that the current exclusion list for ALL tests is currently:

EXCLUDE_ESTIMATORS = [
    "ShapeDTW",
    "ElasticEnsemble",
    "ProximityForest",
    "ProximityStump",
    "ProximityTree",
]

so distances arent really in the greatest shape either way

**Comment by TonyBagnall** [comment]
That explains why the clustering ones are the only fails!

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by TonyBagnall** [comment]
seems warn() works and passes the test, could you review @MatthewMiddlehurst and @mloning?

**Review by MatthewMiddlehurst — APPROVED** [review]
Not sure about todos in the header comments and it doesn't seem the data_io functions have been moved yet.

Nothing that would block this though.

File: sktime/utils/data_io.py [hunk 1]
```diff
@@ -55,6 +59,10 @@ class values.
         all time-series and (if relevant) a column "class_vals" the
         associated class values.
     """
+    warn(
```
Comment (mloning) [inline sktime/utils/data_io.py:62]: can we make this a future warning `warn("msg", FutureWarning)`

Comment (mloning) [reply sktime/utils/data_io.py:62]: and the other warn call below

**Review by mloning — COMMENTED** [review]
Just one minor comment, everything else looks good

**Review by MatthewMiddlehurst — APPROVED** [review]

**Pull request merged**